echopype/convert/convert.py

```diff
@@ -79,8 +99,14 @@ class Convert:
         # get environment XML only (EK80)
         ec.to_netcdf(data_type='ENV_XML')
     """
-
     def __init__(self, file=None, xml_path=None, model=None, storage_options=None):
+        warnings.warn(
+            "Calling `echopype.Convert` directly will be deprecated, "
+            "use `echopyp.conversion.open_raw(file, model, ...)` instead.",
```

Comment:
Change to `echopype.convert.open_raw(file, model, ...)`

Reply:
Ah great! thanks for catching this!